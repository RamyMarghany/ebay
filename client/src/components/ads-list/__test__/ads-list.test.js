
import React from 'react';
import ReactDOM from 'react-dom'
import Ads from './../index'
import "@testing-library/jest-dom"; 
import renderer from 'react-test-renderer'

it('render without crashing', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Ads></Ads>, div)
})


it('matches snapshots', ()=>{
    const tree = renderer.create(<Ads/>).toJSON()
    expect(tree).toMatchSnapshot();
})