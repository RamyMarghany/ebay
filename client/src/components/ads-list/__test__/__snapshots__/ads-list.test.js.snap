// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<div
  className="container"
>
  <div
    className="row"
    style={
      Object {
        "margin": "1.25rem 0 2rem",
      }
    }
  >
    <h2>
      Classifieds
    </h2>
  </div>
  <div
    className="row"
  />
</div>
`;
